toucan_connectors/redshift/redshift_database_connector.py

```diff
@@ -0,0 +1,88 @@
+from contextlib import suppress
+
+import pandas as pd
+import redshift_connector
+from pydantic import Field, SecretStr, create_model
+from pydantic.types import constr
+
+from toucan_connectors.toucan_connector import ToucanConnector, ToucanDataSource, strlist_to_enum
+
+DATABASE_QUERY = """select datname from pg_database;"""
+
+
+class RedshiftDataSource(ToucanDataSource):
+    database: str = Field(..., description='The name of the database you want to query')
+    table: str = Field(None, description='The name of the data table that you want to ')
+    query: constr(min_length=1) = Field(
+        description='A string describing a query (CAUTION: Use limit to avoid to retrieve too many datas)',
+        widget='sql',
+    )
+
+    @classmethod
+    def get_form(cls, connector: 'RedshiftConnector', current_config):
```

Comment:
This function needs to be updated. 
It prevents the data source form to be displayed.